lib/wibox/widget/imagebox.lua

```diff
@@ -307,16 +341,27 @@ end
 -- @method set_image
 -- @hidden
 -- @tparam image image The image to render.
+-- @tparam[opt=nil] table|nil args
+-- @tparam[opt=false] boolean args.force Reload the image even if the path has
+--  not changed.
```

Comment:
I don't think it makes sense to implement this `force` parameter for all values of `image`. For anything but a path, there will be no change, and the resulting signals would be misleading ("emitted when the image changes").
Given that this is about being able to load from a path uncached, just make it explicit:

```suggestion
-- @tparam[opt=false] boolean args.uncached Bypass the cache when loading the image.
```

Either way, you're currently still not bypassing `rsvg_handle_cache`.